maven/tests/paths_test.bzl

```diff
@@ -0,0 +1,13 @@
+load(":testing.bzl", "asserts", "test_suite")
+load("//maven:utils.bzl", "paths")
+
+def filename_test(env):
+    asserts.equals(env, "blah", paths.filename("//foo/bar/blah"))
+    asserts.equals(env, "blah", paths.filename("bar/blah"))
+    asserts.equals(env, "blah", paths.filename("/bar/blah"))
+    asserts.equals(env, "blah", paths.filename("../bar/blah"))
+    asserts.equals(env, "blah", paths.filename("@maven//bar/blah"))
+
+# Roll-up function.
+def paths_test_suite():
+    test_suite("paths", tests = [filename_test]
```

Comment:
Missing `)`

Reply:
Looks like next step is a travis build :)

Reply:
Wait... how did that even parse? o.O

Reply:
Yeesh.  Travis build is right.  Whoops. I thought I had run it before checking that last bit in. :/

Reply:
Ok - Travis added to this PR, because why not.